paper-drawer-panel.html

```diff
@@ -493,6 +497,23 @@
           },
 
           /**
+           * The selector for the element that should receive focus when the drawer is open.
+           */
```

Comment:
Need to add more doc to make sure is clear to the users that this is for element that is in the lightdom of paper-drawer-panel. e.g.
```
<paper-drawer-panel>
  <div drawer>
    <a href="#home">Home</a>
    <a href="#favorites">Favorites</a>
  </div>
  <div> main content...</div>
</paper-drawer-panel>
```

And if the tabbable elements are inside Shadow DOM of some elements, then we can show them how to workaround this by calling focus() programmatically after the drawer is opened.